cropper/app/lib/AspectRatio.scala

```diff
@@ -0,0 +1,21 @@
+package lib
+
+object AspectRatio {
+  case class Ratio(friendly: Option[String], width: Int, height: Int)
+
+  def gcd(a: Int, b: Int): Int = if (b == 0) a else gcd(b, a % b)
+
+  def calculate(width: Int, height: Int) : Option[Ratio] = {
+    val knownRatios = List(
+      Ratio(Some("5:3"), 5, 3),
+      Ratio(Some("2:3"), 2, 3),
+      Ratio(Some("16:9"), 16, 9)
+    )
```

Comment:
You could move that `val` outside the function so it doesn't get allocated every time, but just once.